src/renderer/main.js

```diff
@@ -93,6 +95,8 @@ library.add(
   // solid icons
   faAngleDown,
   faAngleUp,
+  faAnglesDown,
+  faAnglesUp,
```

Comment:
```suggestion
  faAngleDown,
  faAnglesDown,
  faAnglesUp,
  faAngleUp,
```
Alphabetized (see comment at top of file)